backend/src/resolvers/language.resolver.ts

```diff
@@ -0,0 +1,69 @@
+import {Arg, Mutation, Query, Resolver} from "type-graphql";
+
+import Language from "../entities/language";
+
+import DataSource from "../db";
+
+@Resolver(Language)
+
+export default class LanguageResolver {
+    languageRepository = DataSource.getRepository(Language);
+
+    constructor() {}
+
+    // get all languages
+    @Query(() => [Language])
+    async getLanguages() {
+
+        return this.languageRepository.find();
+    }
+
+    // get language by id
+    @Query(() => [Language])
+    async getLanguage(@Arg("id") id: string) {
+
+        return this.languageRepository.findOne({ where: { id } });
```

Comment:
```suggestion
        return this.languageRepository.findOneOrFail({ where: { id } });
```
c'est mieux pour le catch des erreurs, juste la doc de typeorm c'est de la merde